multi/imxrt-multi/config.h

```diff
@@ -766,6 +766,63 @@
 #endif
 #endif
 
+#if !ISEMPTY(UART_CONSOLE)
+#if UART_CONSOLE <= 0
```

Comment:
These two ifs may be squashed into one

Reply:
They can't. If `UART_CONSOLE` is empty, the expression `!ISEMPTY(UART_CONSOLE) && (UART_CONSOLE <= 0)` will result in an error `operator '<=' has no left operand`.